runbooks/runbook.md

```diff
@@ -0,0 +1,112 @@
+# UPP - Annotations Mapper
+
+Annotations Mapper reads all QMI native metadata messages from the `NativeCmsMetadataPublicationEvents` queue and maps them to a JSON message, which is forwarded to the `ConceptAnnotations` queue.
+
+## Code
+
+annotations-mapper
+
+## Primary URL
+
+<https://github.com/Financial-Times/annotations-mapper>
+
+## Service Tier
+
+Platinum
+
+## Lifecycle Stage
+
+Production
+
+## Delivered By
+
+content
+
+## Supported By
+
+content
+
+## Known About By
+
+- dimitar.terziev
+- elitsa.pavlova
+- kalin.arsov
+- ivan.nikolov
+- miroslav.gatsanoga
+- marina.chompalova
+- hristo.georgiev
+- donislav.belev
+- mihail.mihaylov
+- boyko.boykov
+
+## Host Platform
+
+AWS
+
+## Architecture
+
+None provided
```

Comment:
I'd recommend using the current description here, and adding a more general one in place of the description.